certificate-service/app/src/test/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/ag7804/elements/QuestionAktivitetsbegransningarTest.java

```diff
@@ -0,0 +1,141 @@
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.ElementId;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.ElementRuleExpression;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.ElementRuleType;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.IcfCodesPropertyType;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.RuleExpression;
+import se.inera.intyg.certificateservice.domain.validation.model.ElementValidationIcfValue;
+
+class QuestionAktivitetsbegransningarTest {
+
+  @Test
+  void shouldIncludeId() {
+    final var element = QuestionAktivitetsbegransningar.questionAktivitetsbegransningar();
+    assertEquals(QUESTION_AKTIVITETSBEGRANSNING_ID, element.id());
+  }
+
+  @Test
+  void shouldIncludeConfiguration() {
+    final var expectedConfiguration = ElementConfigurationIcf.builder()
+        .id(QUESTION_AKTIVITETSBEGRANSNING_FIELD_ID)
+        .name(
+            "Beskriv vad du bedömer att patienten har svårt att göra på grund av sin sjukdom. Ange exempel på sådana begränsningar relaterade till de arbetsuppgifter eller annan sysselsättning som du bedömer arbetsförmågan i förhållande till. Ange om möjligt svårighetsgrad.")
+        .modalLabel("Välj enbart de svårigheter som påverkar patientens sysselsättning.")
+        .collectionsLabel(
+            "Svårigheter som påverkar patientens sysselsättning:")
+        .placeholder(
+            "Hur begränsar ovanstående patientens sysselsättning och i vilken utsträckning?")
+        .icfCodesPropertyName(IcfCodesPropertyType.AKTIVITETSBEGRANSNINGAR)
+        .build();
+
+    final var element = QuestionAktivitetsbegransningar.questionAktivitetsbegransningar();
+
+    assertEquals(expectedConfiguration, element.configuration());
+  }
+
+  @Test
+  void shouldIncludeRules() {
+    final var element = QuestionAktivitetsbegransningar.questionAktivitetsbegransningar();
+    final var expectedRules = List.of(
+        ElementRuleExpression.builder()
+            .id(QUESTION_AKTIVITETSBEGRANSNING_ID)
+            .type(ElementRuleType.MANDATORY)
+            .expression(
+                new RuleExpression(
+                    "$17.1"
+                )
+            )
+            .build(),
+        ElementRuleExpression.builder()
+            .id(new ElementId("27"))
+            .type(ElementRuleType.HIDE)
+            .expression(new RuleExpression("$27.1"))
+            .build()
+    );
+    assertEquals(expectedRules, element.rules());
+  }
+
+  @Test
+  void shouldIncludeValidation() {
+    final var element = QuestionAktivitetsbegransningar.questionAktivitetsbegransningar();
+    final var expectedValidations = List.of(
+        ElementValidationIcfValue.builder()
+            .mandatory(true)
+            .limit(4000)
+            .build()
+    );
+    assertEquals(expectedValidations, element.validations());
+  }
+
+//  @Test
```

Comment:
We can remove these tests we won't have a pdf config for this since it's general print function